go/plugins/dotprompt/dotprompt.go

```diff
@@ -85,24 +86,20 @@ type Config struct {
 	ModelName string
 
 	// The Model to use.
-	// If this is non-nil, Model should be the empty string.
-	Model ai.Model
+	// If this is non-nil, ModelName should be the empty string.
+	Model *ai.Model
```

Comment:
I'm sorry, I misled you here. I didn't realize that ai.Model is an interface so we actually don't want it to be a pointer. Interfaces can be nil.